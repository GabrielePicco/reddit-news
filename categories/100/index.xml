<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>100 on Reddit News</title><link>https://gabrielepicco.github.io/reddit-news/categories/100/</link><description>Recent content in 100 on Reddit News</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Fri, 31 Dec 2021 15:10:15 +0000</lastBuildDate><atom:link href="https://gabrielepicco.github.io/reddit-news/categories/100/index.xml" rel="self" type="application/rss+xml"/><item><title>Paraguay now produces 100% renewable electric energy</title><link>https://gabrielepicco.github.io/reddit-news/p/61cf1b10f69443c8e0b39905/</link><pubDate>Fri, 31 Dec 2021 15:10:15 +0000</pubDate><guid>https://gabrielepicco.github.io/reddit-news/p/61cf1b10f69443c8e0b39905/</guid><description>Itaipu board member Gerardo Blanco said that Paraguay is moving towards the achievement of a historic triple 100%.
Summary RIO DE JANEIRO, BRAZIL - &amp;ldquo;This is an unprecedented historic event,&amp;rdquo; Abdo Benítez said during the inauguration ceremony. He also said that the inclusion of Bahía Negra into the basic service comes after &amp;ldquo;years of negligence, lack of vision and investment in the electricity system.&amp;rdquo; The head of state highlighted that with the electrification of the city of Bahía Negra, Paraguay becomes the region&amp;rsquo;s first country to have 100% renewable electricity supply.</description></item></channel></rss>