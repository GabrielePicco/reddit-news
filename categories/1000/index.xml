<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>1000 on Reddit News</title><link>https://www.redditnews.io/categories/1000/</link><description>Recent content in 1000 on Reddit News</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Tue, 29 Mar 2022 15:14:56 +0000</lastBuildDate><atom:link href="https://www.redditnews.io/categories/1000/index.xml" rel="self" type="application/rss+xml"/><item><title>Texas governor declares state of disaster as wildfire burns over 1,000 acres</title><link>https://www.redditnews.io/p/6243224cface9c8f41a7c25b/</link><pubDate>Tue, 29 Mar 2022 15:14:56 +0000</pubDate><guid>https://www.redditnews.io/p/6243224cface9c8f41a7c25b/</guid><description>As wildfires continue to blaze through central Texas, one particular fired named Das Goat Fire has prompted Gov. Greg Abbott to declare a state of disaster.
Highlights Das Goat Fire has burned over 1,000 acres in Medina County, Texas. Gov. Greg Abbott said the fire poses an imminent threat of severe damage. Three homes have been lost and 37 others have been threatened, according to the governor. Many areas throughout Texas will remain under a high to extreme elevated fire risk.</description></item></channel></rss>