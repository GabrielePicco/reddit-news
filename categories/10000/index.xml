<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>10000 on Reddit News</title><link>https://www.redditnews.io/categories/10000/</link><description>Recent content in 10000 on Reddit News</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Mon, 02 May 2022 01:01:52 +0000</lastBuildDate><atom:link href="https://www.redditnews.io/categories/10000/index.xml" rel="self" type="application/rss+xml"/><item><title>U.S. Marshals offering $10,000 for missing Alabama inmate and officer</title><link>https://www.redditnews.io/p/626f2d5f875ba6b6bea16cda/</link><pubDate>Mon, 02 May 2022 01:01:52 +0000</pubDate><guid>https://www.redditnews.io/p/626f2d5f875ba6b6bea16cda/</guid><description>The U.S.
Highlights The U.S. Marshals Service is offering up to $10,000 for information leading to the capture of Casey White, a murder suspect. Vicky White, 38, a Lauderdale County corrections officer, left a detention center around 9:40 a.m. Saturday to go to a courthouse but never arrived. Casey White was charged with two counts of capital murder in September 2020 in the stabbing of Connie Ridgeway, 58, and he was already in jail for a series of crimes he committed in 2015, including home invasion, carjacking and a police chase.</description></item></channel></rss>