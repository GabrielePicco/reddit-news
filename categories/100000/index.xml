<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>100000 on Reddit News</title><link>https://www.redditnews.io/categories/100000/</link><description>Recent content in 100000 on Reddit News</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Wed, 29 Dec 2021 09:10:18 +0000</lastBuildDate><atom:link href="https://www.redditnews.io/categories/100000/index.xml" rel="self" type="application/rss+xml"/><item><title>1 in 50 Manhattan Residents Infected With COVID in the Last Week</title><link>https://www.redditnews.io/p/61cc23ab8e1203a6f17935ee/</link><pubDate>Wed, 29 Dec 2021 09:10:18 +0000</pubDate><guid>https://www.redditnews.io/p/61cc23ab8e1203a6f17935ee/</guid><description>The omicron variant outbreak of COVID-19 in New York City is growing so quickly, 2% of all Manhattan residents were infected in just the last week.
Summary As the omicron variant of COVID-19 continues to spread out of control in New York City, transmission rates are soaring, with 2% of all Manhattan residents infected in the last week and nearing that level in the other boroughs. About 2,012 of every 100,000 Manhattan residents tested positive in the last seven days, according to the latest transmission data posted by the city.</description></item></channel></rss>