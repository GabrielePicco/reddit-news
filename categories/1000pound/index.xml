<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>1000pound on Reddit News</title><link>https://www.redditnews.io/categories/1000pound/</link><description>Recent content in 1000pound on Reddit News</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Sun, 08 May 2022 20:14:26 +0000</lastBuildDate><atom:link href="https://www.redditnews.io/categories/1000pound/index.xml" rel="self" type="application/rss+xml"/><item><title>There's a 1,000-pound great white shark swimming near the Jersey shore</title><link>https://www.redditnews.io/p/6278248125d2a2d88aa1285f/</link><pubDate>Sun, 08 May 2022 20:14:26 +0000</pubDate><guid>https://www.redditnews.io/p/6278248125d2a2d88aa1285f/</guid><description>A massive great white shark has been recorded swimming off the New Jersey shore.
Highlights The shark is named Ironbound, after the West Ironbound Island near Lunenburg, Nova Scotia, where it was first spotted. The creature is 12 feet, 4 inches long and weighs an estimated 998 pounds. He pinged in the waters off the coast of New Jersey about 10:30 p.m. on April 28. Great white sharks in the Atlantic Ocean make this migration each year, spending their summers in the northern waters near Canada and their winters as far south as the eastern Gulf of Mexico.</description></item></channel></rss>