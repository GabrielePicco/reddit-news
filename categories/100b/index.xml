<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>100b on Reddit News</title><link>https://www.redditnews.io/categories/100b/</link><description>Recent content in 100b on Reddit News</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Wed, 22 Dec 2021 04:10:06 +0000</lastBuildDate><atom:link href="https://www.redditnews.io/categories/100b/index.xml" rel="self" type="application/rss+xml"/><item><title>Secret Service: Nearly $100B stolen in pandemic relief funds</title><link>https://www.redditnews.io/p/61c2a2d659180f685a50b3e9/</link><pubDate>Wed, 22 Dec 2021 04:10:06 +0000</pubDate><guid>https://www.redditnews.io/p/61c2a2d659180f685a50b3e9/</guid><description>Description The U.S. Secret Service says nearly $100 billion has been stolen from COVID-19 relief programs set up to help businesses and people who lost their jobs due to the pandemic.
Summary Nearly $100 billion at minimum has been stolen from COVID-19 relief programs set up to help businesses and people who lost their jobs due to the pandemic, the U.S. Secret Service said Tuesday. The estimate is based on Secret Service cases and data from the Labor Department and the Small Business Administration, said Roy Dotson, the agencyâ€™s national pandemic fraud recovery coordinator, in an interview.</description></item></channel></rss>