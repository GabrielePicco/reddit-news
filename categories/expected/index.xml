<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>expected on Reddit News</title><link>https://www.redditnews.io/categories/expected/</link><description>Recent content in expected on Reddit News</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Sun, 24 Apr 2022 12:21:04 +0000</lastBuildDate><atom:link href="https://www.redditnews.io/categories/expected/index.xml" rel="self" type="application/rss+xml"/><item><title>US Secretary of State Antony Blinken will visit Kyiv on Sunday - Zelensky</title><link>https://www.redditnews.io/p/626540912723ebe1cce58c71/</link><pubDate>Sun, 24 Apr 2022 12:21:04 +0000</pubDate><guid>https://www.redditnews.io/p/626540912723ebe1cce58c71/</guid><description>The US secretary of state, and the defence secretary, are expected in Ukraine later, President Zelensky says.
Highlights Two residential blocks were hit, killing at least eight people, including a three-month-old, her mother and grandmother. The Ukrainian government has said the killing of baby Kira and the seven other victims is an act of terror, designed to break people’s will to fight. The Russians claim they hit legitimate military facilities.</description></item><item><title>Consumer prices rose 8.5% in March, slightly hotter than expected</title><link>https://www.redditnews.io/p/625588ec6bc4da7578a98836/</link><pubDate>Tue, 12 Apr 2022 14:13:35 +0000</pubDate><guid>https://www.redditnews.io/p/625588ec6bc4da7578a98836/</guid><description>The consumer price index for March was expected to increase 8.4% from a year ago, according to Dow Jones estimates.
Highlights Consumer price index jumped 8.5% from a year ago on an unadjusted basis. Core inflation was the hottest since August 1982. The data reflected price rises not seen in the U.S. since the stagflation days of the late 1970s and early &amp;rsquo;80s. However, core inflation appeared to be ebbing, rising 0.</description></item><item><title>A convoy of truckers protesting Covid-19 measures is expected to arrive in the DC area this weekend. Here's what we know</title><link>https://www.redditnews.io/p/6222c5b17bcaa8c6dd971923/</link><pubDate>Sat, 05 Mar 2022 02:07:15 +0000</pubDate><guid>https://www.redditnews.io/p/6222c5b17bcaa8c6dd971923/</guid><description>Convoy groups led by American truckers are making their way toward the nation&amp;rsquo;s capital to demand an end to all Covid-19 mandates and restrictions.
Highlights In late January, Canadian protesters spent three weeks demonstrating against Covid-19 mandates, crippling three US border crossings and downtown Ottawa. Police arrested more than 100 people when the so-called “Freedom Convoy,” turned violent after truckers blocked roads and irritated residents with honking horns. The protestors are expected to arrive in Washington, DC throughout the upcoming weekend.</description></item></channel></rss>